build-system/tasks/release/index.js

```diff
@@ -151,17 +152,25 @@ function discoverDistFlavors_() {
     ...experimentConfigDefs
       .filter(
         // Only include experiments that have a `define_experiment_constant` field.
-        ([, experimentConfig]) => experimentConfig.define_experiment_constant
+        ([, experimentConfig]) =>
+          'define_experiment_constant' in experimentConfig &&
+          experimentConfig.define_experiment_constant
```

Comment:
Why is `in` needed here?

Reply:
Because now `experimentConfig` can be either `ExperimentConfigDef` or `{}` - creating a type error without the `in`